src/template/js/sdk/Transport/index.js

```diff
@@ -42,13 +43,26 @@ if (!win.__firebolt) {
   win.__firebolt = {}
 }
 
+const _endpoint = () => {
+  const endpoint = win.location ? new URLSearchParams(win.location.search).get('__firebolt_endpoint') : null
```

Comment:
What's the use case for overriding the endpoint via launch URL query params? I'm a bit concerned about this being abused, but it's only a vague concern, not a specific one yet.